### PR TITLE
Introduce gen_ai.request.seed as needed by cohere

### PR DESCRIPTION
**Pull request by xrmx**
## Changes

And deprecate gen_ai.openai.request_seed.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by karthikscale3 — APPROVED** [review]
good call

**Review by karthikscale3 — APPROVED** [review]

File: model/gen-ai/deprecated/registry-deprecated.yaml [hunk 1]
```diff
@@ -28,3 +28,13 @@ groups:
...
         examples: ["[{'role': 'assistant', 'content': 'The capital of France is Paris.'}]"]
+  - id: registry.gen_ai.openai.deprecated
+    type: attribute_group
+    brief: Describes deprecated `gen_ai.openai` attributes.
+    display_name: Deprecated OpenAI GenAI Attributes
+    attributes:
+      - id: gen_ai.openai.request.seed
+        stability: experimental
+        type: int
+        brief: "Deprecated, use `gen_ai.request.seed`."
```
Comment (lmolkova) [inline model/gen-ai/deprecated/registry-deprecated.yaml:39]: please add 

```suggestion
        deprecated: "Replaced by `gen_ai.request.seed`."
```

that's what makes attribute deprecated

Comment (xrmx) [reply model/gen-ai/deprecated/registry-deprecated.yaml:40]: Added and rebuilt tables, thanks!

**Review by trisch-me — APPROVED** [review]

**Review by lmolkova — APPROVED** [review]

**Review by codefromthecrypt — COMMENTED** [review]
thanks, the less one-offs the better!

**Comment by energuminosatlantis-dev** [comment]
Chek mate

**Pull request merged**